--- urbackup-server-2.0.38/urbackupserver/server_settings.cpp.orig	2016-12-13 05:08:05.000902100 +0000
+++ urbackup-server-2.0.38/urbackupserver/server_settings.cpp	2016-12-13 05:14:33.470896941 +0000
@@ -258,12 +258,12 @@
 	settings->no_file_backups=(settings_default->getValue("no_file_backups", "false")=="true");
 	settings->overwrite=false;
 	settings->allow_overwrite=(settings_default->getValue("allow_overwrite", "true")=="true");
-	settings->backupfolder=trim(settings_default->getValue("backupfolder", "C:\\urbackup"));
+	settings->backupfolder=trim(settings_default->getValue("backupfolder", "/mnt/urbackup"));
 	settings->backupfolder_uncompr=trim(settings_default->getValue("backupfolder_uncompr", settings->backupfolder));
 	settings->autoshutdown=(settings_default->getValue("autoshutdown", "false")=="true");;
 	settings->startup_backup_delay=settings_default->getValue("startup_backup_delay", 0);
-	settings->download_client=(settings_default->getValue("download_client", "true")=="true");
-	settings->autoupdate_clients=(settings_default->getValue("autoupdate_clients", "true")=="true");
+	settings->download_client=(settings_default->getValue("download_client", "false")=="true");
+	settings->autoupdate_clients=(settings_default->getValue("autoupdate_clients", "false")=="true");
 	settings->backup_window_incr_file=settings_default->getValue("backup_window_incr_file", "1-7/0-24");
 	settings->backup_window_full_file=settings_default->getValue("backup_window_full_file", "1-7/0-24");
 	settings->backup_window_incr_image=settings_default->getValue("backup_window_incr_image", "1-7/0-24");
@@ -314,7 +314,7 @@
 	settings->internet_calculate_filehashes_on_client=(settings_default->getValue("internet_calculate_filehashes_on_client", "true")=="true");
 	settings->use_incremental_symlinks=(settings_default->getValue("use_incremental_symlinks", "true")=="true");
 	settings->internet_connect_always=(settings_default->getValue("internet_connect_always", "false")=="true");
-	settings->show_server_updates=(settings_default->getValue("show_server_updates", "true")=="true");
+	settings->show_server_updates=(settings_default->getValue("show_server_updates", "false")=="true");
 	settings->server_url=settings_default->getValue("server_url", "");
 	settings->verify_using_client_hashes=(settings_default->getValue("verify_using_client_hashes", "false")=="true");
 	settings->internet_readd_file_entries=(settings_default->getValue("internet_readd_file_entries", "true")=="true");
