--- urbackup-server-2.0.38/Makefile.in.orig	2016-12-13 14:36:19.687449325 +0000
+++ urbackup-server-2.0.38/Makefile.in	2016-12-13 14:39:20.488446924 +0000
@@ -80,7 +80,7 @@
 build_triplet = @build@
 host_triplet = @host@
 sbin_PROGRAMS = urbackupsrv$(EXEEXT)
-bin_PROGRAMS = urbackupsrv$(EXEEXT) urbackup_snapshot_helper$(EXEEXT)
+bin_PROGRAMS = urbackup_snapshot_helper$(EXEEXT)
 @WITH_URLPLUGIN_TRUE@am__append_1 = urlplugin/dllmain.cpp urlplugin/pluginmgr.cpp urlplugin/UrlFactory.cpp
 @WITH_CRYPTOPLUGIN_TRUE@am__append_2 = cryptoplugin/dllmain.cpp cryptoplugin/AESDecryption.cpp cryptoplugin/CryptoFactory.cpp cryptoplugin/pluginmgr.cpp cryptoplugin/AESEncryption.cpp cryptoplugin/ZlibCompression.cpp cryptoplugin/ZlibDecompression.cpp cryptoplugin/AESGCMDecryption.cpp cryptoplugin/AESGCMEncryption.cpp cryptoplugin/ECDHKeyExchange.cpp
 @EMBEDDED_CRYPTOPP_TRUE@@WITH_CRYPTOPLUGIN_TRUE@am__append_3 = cryptoplugin/cryptlib.cpp \
