--- urbackup-server-2.0.38/Makefile.in.orig	2016-12-11 19:57:19.976926847 +0000
+++ urbackup-server-2.0.38/Makefile.in	2016-12-11 20:15:38.746917767 +0000
@@ -1254,7 +1254,7 @@
 	urbackupserver/serverinterface/backups.h \
 	urbackupserver/server_continuous.h urbackupcommon/change_ids.h \
 	urbackupcommon/TreeHash.h $(cryptopp_headers)
-EXTRA_DIST = docs/urbackupsrv.1 init.d_server defaults_server logrotate_urbackupsrv urbackup-server.service urbackup-server-firewalld.xml urbackup/status.htm urbackupserver/www/js/*.js urbackupserver/www/*.htm urbackupserver/www/css/*.css urbackupserver/www/images/*.png urbackupserver/www/images/*.gif urbackupserver/www/*.ico urbackupserver/urbackup_ecdsa409k1.pub urbackupserver/www/*.swf urbackupserver/www/fonts/* tclap/COPYING tclap/AUTHORS server-license.txt
+EXTRA_DIST = docs/urbackupsrv.1 init.d_server defaults_server logrotate_urbackupsrv urbackup-server.service urbackup-server-firewalld.xml urbackup/status.htm urbackupserver/www/js/*.js urbackupserver/www/*.htm urbackupserver/www/css/*.css urbackupserver/www/images/*.png urbackupserver/www/images/*.gif urbackupserver/www/*.ico urbackupserver/www/*.swf urbackupserver/www/fonts/* tclap/COPYING tclap/AUTHORS server-license.txt
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
 
@@ -6416,7 +6416,7 @@
 @FREEBSD_FALSE@@PACKAGING_FALSE@	adduser urbackup --system --quiet --group --home "$(DESTDIR)$(localstatedir)/urbackup" || adduser urbackup --system --user-group --home "$(DESTDIR)$(localstatedir)/urbackup" || true
 @PACKAGING_FALSE@	chown urbackup:urbackup "$(DESTDIR)$(localstatedir)/urbackup"
 
-install-data-local: urbackup/status.htm urbackupserver/www/js/*.js urbackupserver/www/*.htm urbackupserver/www/css/*.css urbackupserver/www/images/*.png urbackupserver/www/images/*.gif urbackupserver/www/fonts/* urbackupserver/urbackup_ecdsa409k1.pub urbackupserver/www/*.swf server-license.txt
+install-data-local: urbackup/status.htm urbackupserver/www/js/*.js urbackupserver/www/*.htm urbackupserver/www/css/*.css urbackupserver/www/images/*.png urbackupserver/www/images/*.gif urbackupserver/www/fonts/* urbackupserver/www/*.swf server-license.txt
 	$(INSTALL) $(INSTALL_OPTS) -m 744 -d "$(DESTDIR)$(localstatedir)/urbackup"
 	chmod +x "$(DESTDIR)$(localstatedir)/urbackup"
 	$(INSTALL) $(INSTALL_OPTS) -m 744 -d "$(DESTDIR)$(datadir)/urbackup"
@@ -6439,10 +6439,8 @@
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/www/images/*.gif "$(DESTDIR)$(datadir)/urbackup/www/images/"
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/www/*.swf "$(DESTDIR)$(datadir)/urbackup/www/"
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/www/fonts/* "$(DESTDIR)$(datadir)/urbackup/www/fonts/"
-	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/urbackup_ecdsa409k1.pub "$(DESTDIR)$(datadir)/urbackup/urbackup_ecdsa409k1.pub"
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/server-license.txt "$(DESTDIR)$(datadir)/urbackup/server-license.txt"
 @PACKAGING_FALSE@	chown urbackup:urbackup "$(DESTDIR)$(datadir)/urbackup"
-@PACKAGING_FALSE@	chown urbackup:urbackup "$(DESTDIR)$(datadir)/urbackup/urbackup_ecdsa409k1.pub"
 @PACKAGING_FALSE@	chown -R urbackup:urbackup "$(DESTDIR)$(datadir)/urbackup/www"
 
 install-exec-hook:
