--- urbackup-server-2.0.38/Makefile.in.orig	2016-12-12 03:44:14.442695337 +0000
+++ urbackup-server-2.0.38/Makefile.in	2016-12-12 03:45:34.930694672 +0000
@@ -6410,6 +6410,7 @@
 @INSTALL_INITD_TRUE@@PACKAGING_FALSE@	if ! test -e "$(sysconfdir)/default/urbackupsrv"; then $(INSTALL) -D "$(srcdir)/defaults_server" "$(sysconfdir)/default/urbackupsrv"; fi
 
 	mkdir -p "$(DESTDIR)$(localstatedir)/urbackup"
+	mkdir -p "$(DESTDIR)$(datadir)/man/man1"
 
 @FREEBSD_TRUE@@PACKAGING_FALSE@	pw groupadd urbackup -q || true
 @FREEBSD_TRUE@@PACKAGING_FALSE@	pw useradd urbackup -q -g urbackup -d "$(DESTDIR)$(localstatedir)/urbackup" -m -s /usr/bin/false || true
@@ -6440,6 +6441,7 @@
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/www/*.swf "$(DESTDIR)$(datadir)/urbackup/www/"
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/urbackupserver/www/fonts/* "$(DESTDIR)$(datadir)/urbackup/www/fonts/"
 	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/server-license.txt "$(DESTDIR)$(datadir)/urbackup/server-license.txt"
+	$(INSTALL_DATA) $(INSTALL_OPTS) $(srcdir)/docs/urbackupsrv.1 "$(DESTDIR)$(datadir)/man/man1/urbackupsrv.1"
 @PACKAGING_FALSE@	chown urbackup:urbackup "$(DESTDIR)$(datadir)/urbackup"
 @PACKAGING_FALSE@	chown -R urbackup:urbackup "$(DESTDIR)$(datadir)/urbackup/www"
 
